src/pages/index.tsx

```diff
@@ -21,6 +24,18 @@ const Auth = () => {
       <Row justify="center">
         <Astronaut />
       </Row>
+      {isAuth && (
+        <Row justify="center">
+          <Button
+            type="default"
+            onClick={() => {
+              push('/signin');
```

Comment:
```suggestion
              push('/graphi');
```
It should be like this